internal/datastore/mysql/datastore.go

```diff
@@ -417,7 +435,24 @@ type querier interface {
 	QueryContext(context.Context, string, ...interface{}) (*sql.Rows, error)
 }
 
-func newMySQLExecutor(tx querier) common.ExecuteQueryFunc {
+type wrappedTX struct {
```

Comment:
why is it wrapped? can we have a better name here?

Reply:
It rewrites it to match the interface expected by the common executor. Any suggestions? `wrappedForCommon`?

Reply:
yeah anything that will indicate the reason for the wrapping, like `withCommonTx` or `asCommonTx` or `commonTx`